src/messages/List/ListRegular.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC } from "react";
+import classes from "./List.module.css";
+import { SingleButton } from "src/common/ActionButtons";
+import { useMessageContext } from "src/hooks";
+import { getBackgroundImage } from "src/lib/css";
+import { useRandomId } from "src/utils";
+import { sanitizeHTML } from "src/sanitize";
+import classnames from "classnames";
+
+const ListRegular: FC<{ element: any }> = props => {
+	const { action, config } = useMessageContext();
+
+	const { title, subtitle, image_url, image_alt_text, buttons, default_action } = props.element;
+
+	const button = buttons && buttons[0];
+
+	const subtitleId = useRandomId("webchatListTemplateSubtitle");
+	const messengerTitle = title ? title + ". " : "";
+	const ariaLabelForTitle = default_action?.url ? messengerTitle + "Opens in new tab" : title;
+
+	const handleKeyDown = (event: any, default_action: any) => {
+		if (default_action && event.key === "Enter") {
+			action?.(event, default_action);
+		}
+	};
+
+	const handleClick = () => {
+		console.log("click");
```

Comment:
remove

Reply:
I will not remove, I am handling the default_action object. I will only trigger the click if is web_url type.

Reply:
done